ibis/backends/snowflake/__init__.py

```diff
@@ -35,9 +36,21 @@
     AlchemyCrossSchemaBackend,
     AlchemyExprTranslator,
 )
-from ibis.backends.snowflake.converter import SnowflakePandasData
-from ibis.backends.snowflake.datatypes import SnowflakeType
-from ibis.backends.snowflake.registry import operation_registry
+
+with warnings.catch_warnings():
+    if vparse(importlib.metadata.version("snowflake-connector-python")) < vparse(
+        "3.3.0"
+    ):
```

Comment:
Don't we want to only ignore the warnings if we're `>=3.3.0`?

Reply:
Yeah, I just discovered that right as you submitted the comment :)